api/data_explorer/dataset_faceted_search.py

```diff
@@ -90,6 +97,19 @@ def get_facets():
     return facets
 
 
+def get_facet_rows():
```

Comment:
This file has the convention that helper methods go before where they're used. Can you move to before get_table_names()?

Reply:
Done